code/game/objects/items/ipc_overloaders.dm

```diff
@@ -53,9 +58,35 @@
 		return
 	to_chat(user, SPAN_WARNING("\The [src] has no use for you!"))
 
+// Jabbing someone else with an overloader.
+/obj/item/ipc_overloader/attack(mob/living/carbon/human/target_mob, mob/user, target_zone)
+	if(isliving(target_mob))
+		user.visible_message(SPAN_WARNING("\The [user] is trying to jab \the [target_mob] with \the [src]!"), SPAN_NOTICE("You are trying to jab \the [target_mob] with \the [src]."))
+
+		if(do_mob(user, target_mob, 2 SECONDS))
+			var/obj/item/organ/external/organ = target_mob.get_organ(target_zone)
+			user.visible_message(SPAN_WARNING("\The [user] jabs \the [target_mob] with \the [src]!"), SPAN_NOTICE("You jab \the [target_mob] with \the [src]."))
```

Comment:
You aren't handling the case where the organ targeted doesn't exist (say, aim a leg that was blown off)